### PR TITLE
chore(deps): update module golang.org/x/net to v0.33.0

### PR DESCRIPTION
**Pull request by codingben**
**What this PR does / why we need it**:

This PR bumps dependencies to resolve [CNV-53634](https://issues.redhat.com//browse/CNV-53634) and [CNV-53627](https://issues.redhat.com//browse/CNV-53627).

**Release note**:

```release-note
None
```

### CODE REVIEW

**Review by 0xFelix — APPROVED** [review]
/lgtm
/approve

**Comment by 0xFelix** [comment]
This line needs to be changed to `apiVersion: ssp.kubevirt.io/v1beta2`

https://github.com/kubevirt/kubevirt-tekton-tasks/blob/87a28d8b0fc748f713335eac01e731cd519a6a23/automation/e2e-deploy-resources.sh#L74

**Comment by codingben** [comment]
@0xFelix @ksimon1 Why it was merged by kubevirt-bot without lgtm label and CI is not passed, after I've added a new commit?

**Pull request merged**
